util/oidc/oidc.go

```diff
@@ -148,32 +139,99 @@ func (a *ClientApp) oauth2Config(scopes []string) (*oauth2.Config, error) {
 	}, nil
 }
 
+func encrypt(data []byte, passphrase string) ([]byte, error) {
+	hasher := md5.New()
+	hasher.Write([]byte(passphrase))
```

Comment:
I'm not sure if we should use `md5` here to derive the key from the passphrase. From the example of the Go docs at https://pkg.go.dev/crypto/cipher#NewGCM, they stress that a cryptographic secure mechanism should be used to derive the key from the passphrase, such as `bcrypt` or `scrypt`. 

If we decide to chose a one-way, non-salted hashing algorithm, I think it should be at least SHA-256 instead of MD5. This would also make the key 32 byte long and use AES-256 instead of AES-128. If we switch to SHA-256, also the follow-up `[]byte(hex.EncodeToString(...))` wouldn't be necessary, and we could just supply `hasher.Sum(nil)` to `aes.NewCipher()` as key.

Reply:
I really like that we can get rid of `[]byte(hex.EncodeToString(...))` with SHA-256 so I've chosen this option